class/blackbox-exporter.yml

```diff
@@ -8,4 +14,13 @@ parameters:
       - input_paths:
           - blackbox-exporter/component/main.jsonnet
         input_type: jsonnet
-        output_path: blackbox-exporter/
+        output_path: blackbox-exporter/${_instance}
```

Comment:
Depending on whether the component should support deploying multiple blackbox exporters in the same namespace (I think this would make sense), you may want to reuse the structure of https://github.com/projectsyn/component-nfs-subdir-external-provisioner/blob/8cdcde392d0d617bb47651a21831afaeeece4671/class/nfs-subdir-external-provisioner.yml#L14-L17 to ensure the namespace manifest is only added once in the final catalog.

Reply:
Fixed